samples/sample-app/views/index.pug

```diff
@@ -37,7 +37,8 @@ html
                     imagekit.upload({
                         file : file.files[0],
                         fileName : file.files[0].name || "test_image.jpg",
-                        tags : ["test_tag_1"]
+                        tags : ["test_tag_1"],
+                        extensions : [{name: "aws-auto-tagging", minConfidence: 80, maxTags: 10}]
```

Comment:
Comment it out so that during testing, user don't end up consuming extension quota.